### PR TITLE
Changes after WGLC

### PR DESCRIPTION
**Pull request by tomaszmrugalski**
These are the changes after WGLC:

- Added references to RFC8987, 6059, 4957.
- Added two items to the implementation report
- updated acknowledgements

### CODE REVIEW

**Comment by bevolz** [comment]
Tomek, wouldn’t the additional text to reference RFC8987 be better at or near end of section 6.3, DHCP for Prefix Delegation?Thanks for starting the work to finish this up.- BernieOn Jan 18, 2024, at 1:03 PM, Tomek Mrugalski ***@***.***> wrote:﻿These are the changes after WGLC. This PR is still work in progress.



You can view, comment on, or merge this pull request online at:
  https://github.com/dhcwg/rfc8415bis/pull/8

Commit Summary

  67f1ec8  reference to RFC8987 (relay-pd requirements) added
  efe20e2  .gitignore tweaked


File Changes  (2 files)

  
    M
    .gitignore
    (1)
  
  
    M
    draft-ietf-dhc-rfc8415bis.xml
    (10)
  


Patch Links:

  https://github.com/dhcwg/rfc8415bis/pull/8.patch
  https://github.com/dhcwg/rfc8415bis/pull/8.diff

**Comment by tomaszmrugalski** [comment]
> Tomek, wouldn’t the additional text to reference RFC8987 be better at or near end of section 6.3, DHCP for Prefix Delegation?

Hey Bernie!
You're right. Moved. Is it any better now?

**Comment by tomaszmrugalski** [comment]
Okay, I went through those handful of comments we received so far and added requested references. There's one discussion thread that sort of hung (See [Esko's mail from Nov. 20](https://mailarchive.ietf.org/arch/msg/dhcwg/frlawaPCzBLbVMUAE7-hElg-qBA/). Esko on Nov. 11 he proposed to update 5 items, but once someone (Tim, I think) explained the scope of bis work, he asked:

> If we change text to clarify something, without proposing new functionality, would that still fit in the 8415bis scope or cleaning up, or not?

but he got no answer. In my opinion the list in 18.2.12 is non-normative. I treat it as a non-exhaustive list of examples, so I'm fine if they want to add couple more. He also proposed to donate a text for "significant change", I'd love to get it.

Thoughts?

**Review by mcr — APPROVED** [review]

**Review by bevolz — APPROVED** [review]
Looks ok to me.

File: draft-ietf-dhc-rfc8415bis.xml [hunk 1]
```diff
@@ -2995,6 +2997,10 @@
             <li>The client changes access points (e.g., if
               using Wi-Fi technology).</li>
           </ul>
+          <t>Specific algorithms for detecting network attachement changes are out of scope
```
Comment (EskoDijk) [inline draft-ietf-dhc-rfc8415bis.xml:3000]: "attachement" -> "attachment"

File: draft-ietf-dhc-rfc8415bis.xml [hunk 2]
```diff
@@ -2995,6 +2997,10 @@
...
+          for this document. Two possible mechanisms for detecting situation where refreshing
```
Comment (EskoDijk) [inline draft-ietf-dhc-rfc8415bis.xml:3001]: "situation" -> "situations"

**Comment by EskoDijk** [comment]
@tomaszmrugalski It looks like my review comments (nits) were missed, or I commented too late. Can you still address these? If not then it's left for the final editor.

File: draft-ietf-dhc-rfc8415bis.xml (hunk 2 above)
Comment (tomaszmrugalski) [reply draft-ietf-dhc-rfc8415bis.xml:3001]: Corrected. Thanks!

**Comment by tomaszmrugalski** [comment]
> @tomaszmrugalski It looks like my review comments (nits) were missed, or I commented too late. Can you still address these? If not then it's left for the final editor.

Yes, I can. Addressed. You wrote on your mail from Jan 25th that you want to send some text next week. I thought I'd just wait and do all of them together.

Anyway, it's not too late.

**Comment by tomaszmrugalski** [comment]
I've pushed those corrections to #9 (see commit 99cc9ffb03c2e5724675a55ea6cee26089636d8b).

**Pull request merged**